docs/troubleshooting-guide/permissions.md

```diff
@@ -0,0 +1,50 @@
+# Fixing permissions issues
+
```

Comment:
When researching this page, was there a reason to include this introductory section? Otherwise, we should cut it/cut to the chase of the issues. Though I would include a further reading section that links out to the resources mentioned here. The reason we should cut it is that we cover this territory more thoroughly elsewhere, and people coming to this page have run into an issue that this introduction is unlikely to illuminate more than the sections below. In its place, we should probably include a table of contents that links to the headings/scenarios.